### PR TITLE
Fixes ear bounty not clarifying it only accepts upgraded cybernetic ears.

### LINKED ISSUE #70495: Bounty for Ears for medical doctors doesn't specify its organic ears
## Reproduction:

the bounty available for medical doctors that asks for ears doesn't specify its organic ears but still won't accept cybernetic ones

for reference there's another bounty called organic eyes which only accepts organic eyes so this bounty should either have its name changed or start accepting cyber ears

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes #70495

:cl: ShizCalev
fix: Fixed ear bounty not clarifying that it only accepts upgraded cybernetic ears.
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
I think you got distracted while writing the title of your PR, Shiz.

**Comment by ShizCalev** [comment]
?

File: code/modules/cargo/bounties/medical.dm [hunk 1]
```diff
@@ -23,13 +23,13 @@
 
 /datum/bounty/item/medical/appendix
 	name = "Appendix"
-	description = "Chef Gibb of Central Command wants to prepare a meal using a very special delicacy: an appendix. If you ship one, he'll pay."
+	description = "Chef Gibb of Central Command wants to prepare a meal using a very special delicacy: an appendix. If you ship one, he'll pay. We'll take a cybernetic one if need be."
```
Comment (Fikou) [inline code/modules/cargo/bounties/medical.dm:26]: there is no cybernetic appendix

File: code/modules/cargo/bounties/medical.dm [hunk 2]
```diff
@@ -23,13 +23,13 @@
...
 	reward = CARGO_CRATE_VALUE * 5 //there are no synthetic appendixes
```
Comment (Fikou) [inline code/modules/cargo/bounties/medical.dm:27]: as said here

File: code/modules/cargo/bounties/medical.dm (hunk 1 above)
Comment (ShizCalev) [inline code/modules/cargo/bounties/medical.dm:26]:
```suggestion
	description = "Chef Gibb of Central Command wants to prepare a meal using a very special delicacy: an appendix. If you ship one, he'll pay."
```

**Comment by ShizCalev** [comment]
@Fikou You know what, I actually thought it was an intentional joke that got removed in #70470 (since why the fuck would anyone ever bother making an artificial appendix.) Looking back even further, seems it was infact just a bad copypaste.

**Pull request merged**